docker-compose.yaml

```
@@ -37,4 +37,10 @@ services:
       MYSQL_USER: shiori
       MYSQL_PASSWORD: shiori
     ports:
-      - "3306:3306"
+      - 3306
+    networks:
+      shiori:
+networks:
```

Adding a new line for looks

```suggestion

networks:
```